[{"user_id": 5108, "stars": [], "topic_id": 34835, "date_created": 1305149651.4873531, "message": "try to import from <youprojectname>.foo.views instead of foo.views", "group_id": 81, "id": 1014799}, {"user_id": 10074, "stars": [], "topic_id": 34835, "date_created": 1305149431.624707, "message": "I just rented a Serevr and set everything up. \nNow I get an import error on every site which needs views of an application. \nViews in the basic app directory work fine. \nWhat are the possible faults I made? \nI just stare at the files for an hour and I can't figure it out...\nThanks for your brainstorming :D", "group_id": 81, "id": 1014759}, {"user_id": 14070, "stars": [], "topic_id": 34835, "date_created": 1305165235.707175, "message": "Make sure foo is in your pythonpath prior to importing from it", "group_id": 81, "id": 1017293}, {"user_id": 10074, "stars": [], "topic_id": 34835, "date_created": 1305188179.9615951, "message": "I thought I always imported from <youprojectname>.foo.views, but in the urls.py i forgot it. Don't know why it works on my local pc. Thanks :)", "group_id": 81, "id": 1022018}, {"user_id": 1127, "stars": [], "topic_id": 34835, "date_created": 1305190026.9057491, "message": "@Samuirai PYTHONPATH works like PATH. If you're in the directory that has the file, your path is auto appended. Otherwise the file has to be on PYTHONPATH", "group_id": 81, "id": 1022474}]